tests/suites/caasadmission/admission.sh

```diff
@@ -57,7 +57,7 @@ EOF
 
 
   # Short sleep to let juju controller watchers catch up.
```

Comment:
\# Medium sleep to ....

Maybe 40s isn't so short anymore :-)

Reply:
Fixed, can lower it now that I am done debugging.